dipy/fixes/scipy.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.linalg import svd
+
+
+__all__ = ['polar']
+
+
+def polar(a, side="right"):
+    """
+    Compute the polar decomposition.
+
+    Returns the factors of the polar decomposition [1]_ `u` and `p` such
+    that ``a = up`` (if `side` is "right") or ``a = pu`` (if `side` is
+    "left"), where `p` is positive semidefinite.  Depending on the shape
+    of `a`, either the rows or columns of `u` are orthonormal.  When `a`
+    is a square array, `u` is a square unitary array.  When `a` is not
+    square, the "canonical polar decomposition" [2]_ is computed.
+
+    Parameters
+    ----------
+    a : (m, n) array_like
```

Comment:
We usually write the shape after the object like in here
http://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.spatial.Delaunay.html
It would be nice if we could follow a standard for this.